docs/DeveloperGuide.md

```diff
@@ -1,8 +1,42 @@
 ---
 layout: page
-title: Travelling BusinessMan, TBM Developer Guide
+title: Travelling BusinessMan(TBM) Developer Guide
```

Comment:
I think we should just stick to either the abbreviation TBM or the full name in the headers, not both

Reply:
either that or perhaps change both back to the simple "User Guide" and "Developer Guide". I don't really see a need to have TBM in the title, as the whole website is already about TBM, plus we have introductions and our home page.

Reply:
done